src/components/AddControlKey.svelte

```diff
@@ -91,31 +90,25 @@
   };
 </script>
 
-<div id="add-control-key" class:hidden={!(connected && showSelf)} class="action-card basis-1/2">
-  <p class="action-card-title">
-    Add a Control Key to Provider Id {providerId}
-  </p>
-  <ol class="ml-4 mt-4 list-decimal font-light">
-    <li>
-      Ensure the new control key has a FRQCY balance if you intend to use it for submitting FRQCY or Capacity
-      transactions.
-    </li>
-    <li>If using a wallet, ensure the new control key is imported into your wallet.</li>
-    <li>Select the new control key from the dropdown list below.</li>
-    <li>Click 'Add It.'</li>
-    <li>This requires 3 signatures: two for the authorization payload, and one to send the transaction.</li>
-    <ul class="ml-6 list-disc">
-      <li>Sign with the new control key,</li>
-      <li>Sign with the current control key,</li>
-      <li>Sign the transaction with the current control key.</li>
-    </ul>
-  </ol>
-  <form class="mt-8">
-    <DropDownMenu id="AddControlKey" label="Key to Add" selected={selectedKeyToAdd} options={validAccounts} />
-    <div class="w-350 flex justify-between">
-      <button on:click|preventDefault={addControlKey} class="btn-primary">Add It</button>
-      <button on:click|preventDefault={cancelAction} class="btn-no-fill">Cancel Add </button>
-    </div>
-  </form>
-</div>
-<TransactionStatus bind:showSelf={showTransactionStatus} statuses={txnStatuses} />
+<Modal id="add-control-key" {isOpen} {close}>
+  <h2 class="section-title-underlined">{`Add Control Key to MSA (${$storeMsaInfo?.msaId})`}</h2>
+
+  <DropDownMenu
+    id="AddControlKey"
+    label="Public Key"
+    selected={$transactionSigningAddress}
```

Comment:
`transactionSigningAddress` is obsolete.  Use `$user.signingKey` instead.  Also, do you need to bind the prop?